.github/workflows/lint.yml

```diff
@@ -85,6 +85,7 @@ jobs:
             sw/math/src/internal/*
             sw/math/src/math/*
             sw/math/Makefile
+            hw/snax/src/*
```

Comment:
Could we not add a license in those files, instead of excluding them from linting?

Reply:
Oh! Okay I'll do that instead. But I think I remember it complains if there is no licensing mentioned. I may be wrong but yeah, sure we can exclude the licensing on our files for now :)